src/styles/main.css

```diff
@@ -0,0 +1,189 @@
+.catalog {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  justify-content: center;
+  gap: 46px 48px;
+}
+@media (max-width: 487px) {
+  .catalog {
+    grid-template-columns: repeat(1, 200px);
+  }
+}
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+@media (min-width: 768px) {
+  .catalog {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+@media (min-width: 1024px) {
+  .catalog {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  transition: all 300ms ease-in-out;
+  border: solid 1px #f3f3f3;
+  width: 200px;
+  height: 408px;
+  background-image: url("../images/imac.jpeg");
+  background-position-y: 32px;
+  background-position-x: 19px;
+  background-size: 160px, 134px;
+  background-repeat: no-repeat;
+}
+.card__text {
+  transition: all 300ms ease-in-out;
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  margin-top: 206px;
+  padding: 0 16px;
+}
+.card__prod-code {
+  color: #616070;
+  font-size: 10px;
+  padding: 4px 16px;
+}
+.card__reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 16px;
+  padding: 0 16px;
+}
+.card__reviews-text {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-left: 17px;
+}
+.card__price {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+  margin-top: 24px;
+  padding: 0 16px;
+  display: flex;
+  flex-direction: row;
+}
+.card__price-value {
+  margin-left: 85px;
+  font-weight: 700;
+  font-size: 16px;
+  color: #000;
+}
+.card__button-buy {
+  transition: all 300ms ease-in-out;
+  width: 166px;
+  height: 40px;
+  border-width: 0;
+  border-radius: 5px;
+  background-color: #00acdc;
+  margin: 16px;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+}
+.card__button-buy:hover {
+  background-color: #fff;
+  color: #00acdc;
+  border: solid 1px #00acdc;
+}
+.card:hover {
+  transform: scale(1.2);
+}
+.card:hover .card__text {
+  color: #34568b;
+}
+
+.card__reviews-star {
+  background-image: url(../images/star-active.svg);
+  width: 16px;
+  height: 16px;
+  background-position: initial;
+  background-repeat: no-repeat;
+}
+
+.card__reviews-star:nth-child(n+5) {
+  background-image: url(../images/star.svg);
+}
```

Comment:
The use of `:nth-child(n+5)` assumes a fixed number of stars. Consider a more flexible approach if the number of stars can vary.